### ISSUE TITLE
Datasets conversion tool

### ISSUE THREAD

**Issue by dikubab** [amazon-research/glass-text-spotting#5]
Can you provide datasets conversion tools?

**Comment by linxid** [comment]
Can you provide icdar2015 dataset conversion script, thanks

**Comment by tsiper** [comment]
Yes definitely, we'll prioritize it and aim to upload it in about a week, I'll keep you posted in any case.

**Comment by WaylonZhangW** [comment]
> 

Hi, may i ask whether you upload the conversion scripts? i cound't find the scripts.

**Comment by linxid** [comment]
> 

Thanks a lot, but now I still can not find converision scripts.

**Comment by tiamjiakun** [comment]
+1
